### PR TITLE
Enable create a new Lambda from explorer

### PR DESCRIPTION
**Pull request by abrooksv**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)

## Description
Enable creating Lambda from the explorer / action menu which will create a blank UI

## Motivation and Context
Now that the gutter icons are changing, we need new entry points to create a new Lambda

## Related Issue(s)
#349

## Testing
Manual testing

## Screenshots (if appropriate)
screen shot 2018-11-01 at 3 07 24 pm
screen shot 2018-11-01 at 3 07 17 pm

## Checklist

- [x] I have read the **README** document
- [x] I have read the **CONTRIBUTING** document
- [x] Local run of `gradlew check` succeeds
- [x] My code follows the code style of this project
- [ ] I have added tests to cover my changes
- [x] All new and existing tests passed
- [ ] A short description of the change has been added to the **CHANGELOG**

## License
I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by abrooksv** [comment]
Should we drop the `...`?

**Comment by kiiadi** [comment]
Wonder that's 'named' correctly. To mean 'Create new AWS Lambda...' is going to help me create the code I need too, in this case we're just packaging local code and deploying it remotely.

Is "Create new remote AWS Lambda..." too long?

File: jetbrains-core/src/software/aws/toolkits/jetbrains/services/lambda/upload/CreateLambdaFunction.kt [hunk 1]
```diff
@@ -9,16 +9,21 @@ import com.intellij.openapi.actionSystem.LangDataKeys
...
-import software.aws.toolkits.jetbrains.services.lambda.orThrow
 import software.aws.toolkits.jetbrains.services.lambda.runtime
 import software.aws.toolkits.resources.message
 
-class UploadLambdaFunction(private val handlerName: String) : AnAction(message("lambda.create_new"), null, AwsIcons.Actions.LAMBDA_FUNCTION_NEW) {
+class CreateLambdaFunction(private val handlerName: String? = null) : AnAction(message("lambda.create_new"), null, AwsIcons.Actions.LAMBDA_FUNCTION_NEW) {
     override fun actionPerformed(event: AnActionEvent) {
         val project = event.getRequiredData(LangDataKeys.PROJECT)
-        val runtime = event.runtime().orThrow
+        val runtime = event.runtime()
```
Comment (kiiadi) [inline jetbrains-core/src/software/aws/toolkits/jetbrains/services/lambda/upload/CreateLambdaFunction.kt]: Can delete the extension function then - that was the only place it was used I believe.

Comment (abrooksv) [reply jetbrains-core/src/software/aws/toolkits/jetbrains/services/lambda/upload/CreateLambdaFunction.kt]: Oh, didnt even look

**Pull request merged**